docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Rerunning introspection"
+date: 2020-06-04T08:14:51-05:00
+draft: true
+weight: 5
+description: "This document describes rerunning the introspector job in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+
+This document describes _how and when_ to rerun the introspector job in the Oracle WebLogic Server in Kubernetes environment.
+
+#### Overview
+
+In order to manage a domain, the operator needs to know its definition in WebLogic Server. It does this by running an _introspector job_
+on the domain definition when the domain is first deployed in the operator. If the configuration is modified, the operator must rerun
```

Comment:
"the operator must rerun the introspector job" implies this automatically happens without administrative intervention.

Reply:
How about, "the customer must trigger the operator to repeat this domain introspection."

Reply:
Or, "you must trigger the operator to repeat this domain introspection."